service/telemetry/config.go

```diff
@@ -103,3 +104,12 @@ type MetricsConfig struct {
 	// Address is the [address]:port that metrics exposition should be bound to.
 	Address string `mapstructure:"address"`
 }
+
+// TracesConfig defines settings to configure context propagation of the collector traces. Collector uses
+// go.opentelemetry.io/contrib/propagators/autoprop package to propagate the context and OTEL_PROPAGATORS
+// environment variable can be set to configure the propagators.
+type TracesConfig struct {
+	// DisablePropagation disables propagation of the trace context to external backend beyond exporters.
+	// The context is propagated by default.
+	DisablePropagation bool `mapstructure:"disable_propagation"`
```

Comment:
I think we should have a propagation section with an enabled field, since (i) this is the pattern we follow elsewhere and (ii) it's possible that we will get other configuration settings related to propagation. So, something like:

```
traces:
  propagator:
    enabled: true
```

I am not familiar with propagation, so maybe the approach should be different (should you have to choose what propagator to use?)

Reply:
I like this structure propagation does have other possible configurations - like (type of propagator etc) - Let me restructure the propagation into its own nested struct and reissue a PR.